### PR TITLE
Starry field

### PR DESCRIPTION
**Pull request by OmiCoding**
Created starry field animation

### CODE REVIEW

**Review by HermanDhillon — APPROVED** [review]

File: controllers/authController.js [hunk 1]
```diff
@@ -72,12 +70,16 @@ exports.postSignup = (req, res, next) => {
     password: req.body.password,
   });
 
+  console.log(req.body);
```
Comment (nmpereira) [inline controllers/authController.js]: console.log? in production?

File: controllers/authController.js [hunk 2]
```diff
@@ -72,12 +70,16 @@ exports.postSignup = (req, res, next) => {
...
+
   User.findOne(
     { $or: [{ email: req.body.email }, { userName: req.body.userName }] },
     (err, existingUser) => {
       if (err) {
         return next(err);
       }
+
+      console.log(existingUser);
```
Comment (nmpereira) [inline controllers/authController.js]: another console.log

**Review by nmpereira — CHANGES_REQUESTED** [review]
console.log is not acceptable

**Review by nmpereira — APPROVED** [review]
perfect!

**Review by Asalz20 — APPROVED** [review]
Css looks phenomenal

**Pull request merged**